### PR TITLE
Topic names

### PR DESCRIPTION
**Pull request by Karsten1987**
connects to ros2/rmw_fastrtps#114

this should fix the inconsistency with `rclcpp::Node::count_subscribers` and `rclcpp::Node::count_publishers` as well as `get_topic_names_and_types`.

The function truncates the internal topic fqdn in the form of `rt/my_namespace/my_topic` into `/my_namespace/my_topic`

### CODE REVIEW

File: rmw_connext_cpp/src/functions.cpp [hunk 1]
```diff
@@ -1878,7 +1878,10 @@ rmw_get_topic_names_and_types(
   rmw_topic_names_and_types_t * topic_names_and_types)
 {
   return get_topic_names_and_types(rti_connext_identifier, node,
-           topic_names_and_types);
+           topic_names_and_types,
+           ros_topics_prefix,
+           ros_service_requester_prefix,
+           ros_service_response_prefix);
```
Comment (mikaelarguedas) [inline rmw_connext_cpp/src/functions.cpp]: please wrap after the open parenthesis and use 2 spaces indentation (same everywhere below)

Comment (Karsten1987) [inline rmw_connext_cpp/src/functions.cpp]: I left it like this on purpose for not blowing up the diff. The style format here should be excluded in a different PR IMO.

Comment (mikaelarguedas) [inline rmw_connext_cpp/src/functions.cpp]: Fair enough.
Would you mind doing it for the cases where the functions were fitting in one line before this PR?

Comment (Karsten1987) [inline rmw_connext_cpp/src/functions.cpp]: I'll open a new PR for style on this.

Comment (mikaelarguedas) [inline rmw_connext_cpp/src/functions.cpp]: :+1:

**Comment by wjwwood** [comment]
Since this is reviewed and tested, and since the connected pr's have already been merged I'm going to merge this for you @Karsten1987.

**Comment by dirk-thomas** [comment]
As of this PR `rmw_connext_dynamic_cpp` fails to build.
